tools/cli/commands/changelog.js

```diff
@@ -235,22 +273,47 @@ async function changelogArgs( argv ) {
 		argv.args.splice( argv.args.indexOf( argv.project ), 1 );
 	}
 
-	// Passthrough arguments for "add"
-	if ( argv.args[ 0 ] === 'add' ) {
-		if ( argv.s && argv.t && argv.e ) {
-			argv.args.push( '--no-interaction' );
-		} else if ( argv.s || argv.t || argv.e ) {
-			console.error(
-				chalk.bgRed(
-					'Need to pass all arguments for non-interactive mode. Defaulting to interactive mode.'
-				)
-			);
-		}
+	// Check for required command specific arguements.
+	switch ( argv.args[ 0 ] ) {
+		case 'add':
+			if ( argv.s && argv.t && argv.e ) {
+				argv.args.push( '--no-interaction' );
+			} else if ( argv.s || argv.t || argv.e ) {
```

Comment:
Is there a reason we're checking specifically for `else if ( argv.s || argv.t || argv.e )` to show the error message here? I'm thinking either someone has passed all required args, or they haven't. If someone does something silly like `jetpack changelog add plugins/jetpack -f test` we should still show that error message before kicking them over to the interactive mode.

Reply:
Yeah, this one is tricky because it will work if you just pass it `-f` - it will rename the default file name in interactive mode with whatever you pass it. Open for a better suggestion of handling that case! 

`jetpack changelog add plugins/jetpack -f test` actually works and does notify it kicks you over to interactive mode.

Reply:
Ah I was just thinking you could do a plain `else` there which would show the error message in cases like that example before passing control to the interactive mode. Not a huge deal, just wasn't sure why we were checking those specific cases in the `else if`.

Reply:
Ah yeah, if you do just the `else` statement there, it also throws the error about non-interactive mode even if you just use `jetpack changelog add` or `jetpack changelog add [project]` (where the user is likely intending interactive mode anyway).

Reply:
Got it, in that case it's probably best to leave as is 👍